src/caret_analyze/plot/bokeh/plot_factory.py

```diff
@@ -15,6 +15,8 @@
 from logging import getLogger
 from typing import Collection, Union
 
+from matplotlib.path import Path
```

Comment:
It looks to me that this line should be remmoved.
Path is defined : https://github.com/tier4/CARET_analyze/blob/main/src/caret_analyze/runtime/__init__.py#L32